lib/codegen/generator.dart

```diff
@@ -292,7 +292,7 @@ class _DeclarationCode extends ConstExpression {
 
   List<String> get librariesUsed => []
     ..addAll(type.librariesUsed)
-    ..addAll(annotations.expand((a) => a.librariesUsed));
+    ..addAll(annotations.expand((a) => a.librariesUsed) as Iterable<String>);
```

Comment:
Actually, it turns out that it does have a generic! ([link](https://github.com/dart-lang/sdk/blob/master/sdk/lib/core/iterable.dart#L186))

You should be able to pass the type to `expand`, for example:
```dart
..addAll(annotations.expand/*<String>*/((a) => a.librariesUsed));
```